### PR TITLE
raftstore/batch: cleanup test code

### PR DESCRIPTION
**Pull request by BusyJay**
## What have you changed? (mandatory)

Reusing runner in batch to clean up test code in router.

## What are the type of the changes? (mandatory)

- Improvement

## How has this PR been tested? (mandatory)

Unit tests.

## Does this PR affect documentation (docs) update? (mandatory)

No.

## Does this PR affect tidb-ansible update? (mandatory)

No.

### CODE REVIEW

File: src/raftstore/store/fsm/router.rs [hunk 1]
```diff
@@ -472,166 +469,136 @@ impl<N: Fsm, C: Fsm, Ns: Clone, Cs: Clone> Clone for Router<N, C, Ns, Cs> {
 }
 
 #[cfg(test)]
-pub mod tests {
+mod tests {
     use super::*;
-    use crossbeam::channel::{SendError, TryRecvError, TrySendError};
+    use crossbeam::channel::{SendError, TrySendError};
+    use raftstore::store::fsm::batch::tests::Message;
+    use raftstore::store::fsm::batch::{self, tests::*};
```
Comment (zhangjinpeng1987) [inline src/raftstore/store/fsm/router.rs]: It's better to move the common part to test/util?

Comment (BusyJay) [reply src/raftstore/store/fsm/router.rs]: No, they are in different crates.

**Comment by BusyJay** [comment]
PTAL

**Pull request merged**